gitlab/v4/objects.py

```diff
@@ -695,6 +695,43 @@ class DeployKeyManager(ListMixin, RESTManager):
     _obj_cls = DeployKey
 
 
+class DeployToken(ObjectDeleteMixin, RESTObject):
+    pass
+
+
+class DeployTokenManager(ListMixin, RESTManager):
+    _path = "/deploy_tokens"
+    _obj_cls = DeployToken
+
+
+class ProjectDeployToken(ObjectDeleteMixin, RESTObject):
+    pass
+
+
+class ProjectDeployTokenManager(ListMixin, CreateMixin, DeleteMixin, RESTManager):
+    _path = "/projects/%(project_id)s/deploy_tokens"
+    _from_parent_attrs = {"project_id": "id"}
+    _obj_cls = ProjectDeployToken
+    _create_attrs = (
+        ("name", "expires_at", "username", "scopes",),
+        tuple(),
```

Comment:
```suggestion
        ("expires_at", "username",),
```

Reply:
As I explained is the description:
```
The managers I added require the four parameters to make tests pass with gitlab/gitlab-ce:nightly.
```
I re-defined them as not required.
It turned out that the API doesn't comply with the spec (the doc), waiting for https://gitlab.com/gitlab-org/gitlab/-/issues/211878 to be fixed.